scripts/install-chef-suse.sh

```diff
@@ -292,12 +300,27 @@ EOF
     exit 1
 fi
 
+if [ -f /opt/dell/crowbar_framework/.crowbar-install-failed ]; then
+    rm /opt/dell/crowbar_framework/.crowbar-install-failed
+    for i in pre_sanity_checks \
+         run_services \
+         initial_chef_client \
+         barclamp_install \
+         bootstrap_crowbar_setup \
+         apply_crowbar_config \
+         transition_crowbar \
+         chef_client_daemon \
+         post_sanity_checks; do
+         [[ -f /opt/dell/.${i} ]] && rm /opt/dell/.${i}
```

Comment:
Ah, actually, you're really putting things directly in /opt/dell. That feels bad, tbh. Can't we just use /var/lib/crowbar instead? And no need to use dot files.